src/background/background.js

```diff
@@ -3,15 +3,15 @@ import Process from './background-process'
 import { Monitor as ExperienceMonitor } from 'alpheios-experience'
 
 // Detect browser features
-let browserFeatures = new Browser().inspect().getFeatures()
-console.log(`Support of a "browser" namespace: ${browserFeatures.browserNamespace}`)
-if (!browserFeatures.browserNamespace) {
+window.browserFeatures = new Browser().inspect().getFeatures()
```

Comment:
why does this need to be global?

Reply:
That's because we were relying on that in `auth0-chrome` where we were having two code lines for FF and Chrome. But we don't have that now, and we can change it back to local. Nice catch!